### PR TITLE
#165876222 Partner stats cards

### PR DESCRIPTION
**Pull request by mikenthiwa**
#### What does this PR do?
- Implements designs for partner stats card.

#### Description of Task to be completed?
- Adding partner stats card on the dashboard.

#### How should this be manually tested?
- Git clone this repository `https://github.com/andela/bp-esa-frontend.git`.
- Checkout to `ft-partner-stats-card-165876222`.
- Run `yarn install` and `yarn start`.
- Navigate to `http://localhost:port/newPage`

#### Any background context you want to provide?
- Dummy routes and values/data used.

#### What are the relevant pivotal tracker stories?
[#165876222](https://www.pivotaltracker.com/story/show/165876222)

#### Screenshots (If applicable)

Screenshot 2019-05-14 at 16 14 35

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -78,6 +79,14 @@ class App extends React.PureComponent {
               currentUser={currentUser}
               removeCurrentUser={this.removeCurrentUser}
             />
+            <AuthenticatedRoute
+              exact
+              path="/newPage"
```
Comment (FelixWambiri) [inline src/App.jsx]: I think that the URL should lead to the dashboard page which will contain the Partners stats card component.

File: src/components/PartnerStatsCard/index.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import Card from '../Dashboard/Card';
+import './partnersStatsCard.scss';
+
+class PartnerStats extends Component {
+  renderStatsCard = (type) => {
+    const renderDiv = value => (
```
Comment (FelixWambiri) [inline src/components/PartnerStatsCard/index.jsx]: This is just a suggestion:  What if you made `renderDiv`  a stand-alone function by itself an call it in the `renderStartsCard` function to improve the readability of your code.

File: src/components/PartnerStatsCard/index.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      <div className="stats-value">{value}</div>
+    );
+    const statsCard = {
+      newPartner: renderDiv(7),
+      partnerChurns: renderDiv(2),
```
Comment (koechkevin) [inline src/components/PartnerStatsCard/index.jsx]: @mikenthiwa good work on matching the mock ups. But quick question is this data expected to be static?

File: src/App.jsx (hunk 1 above)
Comment (mikenthiwa) [reply src/App.jsx]: @FelixWambiri This should change once the dashboard page is ready. It was for testing

File: src/components/PartnerStatsCard/index.jsx (hunk 3 above)
Comment (mikenthiwa) [reply src/components/PartnerStatsCard/index.jsx]: We will read data from props, currently, there is no backend data that is available for this feature.

**Comment by 09arnold** [comment]
@mikenthiwa I removed the test route 'newPage' prior to merging into `develop`

**Pull request merged**